examples/lcd_16x2/lcd_1602.h

```diff
@@ -0,0 +1,116 @@
+/*
+	lcd_1602.h - C implementation of the LiquidCrystal class, providing basic
+	API for communicating with the 16x2 HD44780 LCD; 
+
+	The code was developed based on the basic Arduino Library for the LCD: 
+		https://github.com/arduino-libraries/LiquidCrystal.git
```

Comment:
:(  :(

I don't think we can take this code unfortunately. My understanding is that this is a derivative work of a GPL-licensed driver, and pulling that in would require libtock-c to become GPL as well.

@alevy are actually going to confer with some software licensing professionals next week, so I'll try to ask about this specific case in case I'm wrong, but I think we would need to ask the upstream library to re-license as Apache or MIT or something equally permissive.

Reply:
Unfortunately, I think it's confirmed that we need to either ask the upstream library to re-license a release for us to base a derivative work on or we'll have to reimplement this from scratch from datasheets.